src/components/cards/MyWallet/MyWallet.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <BalCard noPad :hFull="upToLargeBreakpoint" noShadow>
+    <div class="flex flex-col bg-white w-full">
+      <div class="flex justify-between p-3 shadow-lg">
+        <h6>My Wallet</h6>
```

Comment:
Let's get text into the locales as much as possible to make supporting multiple languages in the future easier.

Reply:
yeah was leaving translations to the last moment